apps/api/src/ticle/ticle.controller.ts

```diff
@@ -0,0 +1,19 @@
+import { Controller, Get, Param, Post, Query } from '@nestjs/common';
+
+@Controller('ticle')
+export class TicleController {
+  @Post('')
```

Comment:
p1:
endpoint가 `/`일 경우 빈 값을 넣어도 좋습니다

```suggestion
  @Post()
```